big_tests/tests/gdpr_SUITE.erl

```diff
@@ -154,6 +157,10 @@ init_per_testcase(retrieve_mam = CN, Config) ->
             dynamic_modules:ensure_modules(domain(), mam_required_modules(Backend)),
             escalus:init_per_testcase(CN, Config)
     end;
+init_per_testcase(remove_roster = CN, Config) ->
+    dynamic_modules:ensure_modules(domain(), [{mod_roster, []}]),
+    escalus_fresh:clean(),
```

Comment:
This is usually invoked after test case or after suite.

Reply:
Moved to end per test case.